screens/Map.js

```diff
@@ -83,6 +84,12 @@ export default class Map extends React.Component {
   };
 
   handlePress = marker => {
+    this.regionChange({
+      ...event.nativeEvent.coordinate, 
+      latitudeDelta: 0.005,
+      longitudeDelta: 0.005,
+    })
+    
```

Comment:
Hey @smparelhoff, I added this function in here so that it will reset the region when we click on a marker, instead of just resetting to the user's location. The rerender feels a little choppy so open to other ideas, obviously

Reply:
do we need to pass `event` into `handlePress` as the first param now?

Reply:
also, where is `regionChange` defined?